ncclient/operations/third_party/iosxe/rpc.py

```diff
@@ -1,9 +1,500 @@
 from lxml import etree
-
 from ncclient.xml_ import *
 from ncclient.operations.rpc import RPC
+from ncclient.operations.rpc import RPCReply
+from ncclient.operations.errors import AlreadyHasEventListener
+from ncclient.transport import SessionListener
+from dateutil.parser import parse
+import logging
+
+
+logger = logging.getLogger("ncclient.operations.rpc")
 
 class SaveConfig(RPC):
     def request(self):
         node = etree.Element(qualify('save-config', "http://cisco.com/yang/cisco-ia"))
         return self._request(node)
+
+
+#
+# Message with period specified
```

Comment:
It conforms to either draft 4 or draft 5. This means that it is, in effect, an IOS XE-only implementation. Hence placing it in an IOS XE device handler for now. I will add a comment indicating this, but will not try to identify the specific draft version, as it may not be to a specific draft as the drafts were in flux at the time of coding (in IOS XE).